setup.cfg

```diff
@@ -0,0 +1,27 @@
+[metadata]
+name = git-filter-repo
+url = https://github.com/newren/git-filter-repo
+project_urls = 
+    Source = https://github.com/newren/git-filter-repo
+    Issues = https://github.com/newren/git-filter-repo/issues/
+description = git filter-branch replacement
+version = 2.25.0
+long_description = file: README.md
+author = Elijah Newren
+classifiers =
+    Development Status :: 3 - Alpha
+    Operating System :: OS Independent
+    Programming Language :: Python
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
```

Comment:
I haven't tested with PyPy yet, so I'm hesitant to claim it works with it.  Or is this not a claim that it's tested, just that it should theoretically work with it?

Reply:
Uh, sorry, yeah, that's just another thing my package script autogenerates -- I think the same applies as the above, which is "if you don't use any CPython C extensions in obscene ways, it probably works on PyPy and is safe to put this here", so it's the same "supported" vs "tested" question --

I.e., if someone tried to run this tomorrow on PyPy and found some bug, would you want to investigate it or say "nah that's PyPy and not supported even to investigate" -- if the former, would probably keep this, if the latter, would axe it.

Reply:
Ah, that makes sense -- intent to support.  So, yeah, let's keep PyPy and python >= 3.5.